libyul/backends/evm/EVMDialect.cpp

```diff
@@ -120,6 +120,13 @@ std::set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 		return _instr == evmasm::Instruction::BASEFEE && _evmVersion < langutil::EVMVersion::london();
 	};
 
+	// TODO remove this in 0.9.0. We allow creating functions or identifiers in Yul with the name
```

Comment:
We have exceptions for `basefee`, `blobbasefee` and `prevrandao`, which need to be removed for `0.9.0`. Do we have an issue tracking this?

Reply:
For the `prevrandao` I believe is this one here: https://github.com/ethereum/solidity/issues/13886 but we don't have for the others.

Reply:
#13886 does not mention this hack though. I'm not sure we'd even remember to remove this when fixing it.

But anyway, since this is not the last opcode we're adding, we should just create a catch-all issue for Yul identifiers we're planning to reserve in the next breaking release.

Reply:
Yup, this is why I asked - I wanted to create one before  new years, but was unsure as to whether we already had one laying around.

Reply:
We may have parts of this stuff lingering on the ``breaking`` branch already - but feel free to create issues for this and mark them ``must have eventually`` and ``breaking`` and add them to the 0.9 milestone and such, if you haven't already.

Reply:
[Done](https://github.com/ethereum/solidity/issues/14770).